settings.gradle

```diff
@@ -0,0 +1,7 @@
+include ':fxa-client-library'
+include ':logins-library'
+include ':logins-sample'
+
+project(':fxa-client-library').projectDir = new File("fxa-client/sdks/android/library")
```

Comment:
Can we add docs somewhere on how to build these? 

For example I open Android Studio and it finds the 3 projects, but I know others use the CLI

Reply:
Happy to, although this is pretty standard Gradle stuff.  I'll put it in a README somewhere.